packages/i18n/scripts/import-moment-locales.sh

```diff
@@ -0,0 +1,5 @@
+echo "Importing moment.js locale metadata files."
+mkdir -p ./compiled-data/moment/locales
+cp -R ../../node_modules/moment/dist/locale/*.js ./compiled-data/moment/locales
+# copied files have a relative import to 'moment' so we need to update it to be global
```

Comment:
Imported files look like this:
![image](https://user-images.githubusercontent.com/97907068/177132151-5a87ff33-49cb-42d8-ba3e-66ede0e77bcd.png)

We need to update the import to be:
```
import moment from `moment`;
```
_(`moment` is a direct dependency of this package)_.